### ISSUE TITLE
"Search in Directory" should be in a fixed order in the menu

### ISSUE THREAD

**Issue by gam-phon** [atom/find-and-replace#465]
When clicking on the project "Search in Directory" will be on the bottom above the last one.
![screen shot 2015-07-29 at 12 06 31](https://cloud.githubusercontent.com/assets/1763722/8956510/5a582c96-35ff-11e5-93a9-320523845d9f.png)

When clicking on any folder inside the project "Search in Directory" will be the first one in order.
![screen shot 2015-07-29 at 12 06 45](https://cloud.githubusercontent.com/assets/1763722/8956513/61546a96-35ff-11e5-88c6-8d82c5d6aa60.png)

This confuse me a lot. I do search a lot and I got confused when I want to search some time it is above and other times is down. I think it would be better to be in a fixed place.

**Comment by mnquintana** [comment]
This is currently blocked on https://github.com/atom/atom/issues/6270

**Comment by gam-phon** [comment]
This is a duplicate issue with this https://github.com/atom/find-and-replace/issues/387 So you could close one of them to not get confused later.

Thanks

**Comment by mnquintana** [comment]
Ah thanks! I thought there was a duplicate somewhere. Closing in favor of #387
